packages/client/src/index.ts

```diff
@@ -5,7 +5,7 @@ export enum PenumbraRequestFailure {
 
 export const PenumbraSymbol = Symbol.for('penumbra');
 
-export interface PenumbraProvider {
+export interface PenumbraInjection {
```

Comment:
What's the thinking behind the rename? Is this standard with other wallets? I feel like the `Provider` naming is more common in things I've encountered.

Reply:
`Provider` naming is common for react components that provide something to child components,  which felt awkward while writing the demo and using other `Provider`-named things, so i renamed this.